golang-basic/00-pre-requirements.md

```diff
@@ -0,0 +1,5 @@
+## Pre requirements
+
+1. Tools needed for development: **go1.26**, **iTerm**, **git**, **IntelliJ IDEA**.
+2. IntelliJ IDEA should have next plugins installed:
```

Comment:
I will leave comments for myself to adjust this docs and you can merge in this way to get started with something.

So I guess now that we don't need Gradle and all it's better to tell to use GoLand or Idea with plugins (if they have a paid version). I guess community addition should suffice for them.